MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -231,11 +230,11 @@ - (void)openWithCredentials:(MXCredentials*)someCredentials onComplete:(void (^)
         }
         
         dispatch_async(dispatch_get_main_queue(), ^{
-#if TARGET_OS_IPHONE
-            [[UIApplication sharedApplication] endBackgroundTask:backgroundTaskIdentifier];
-            backgroundTaskIdentifier = UIBackgroundTaskInvalid;
-#endif
-
+            id<MXBackgroundModeHandler> handler = [MXSDKOptions sharedInstance].backgroungModeHandler;
+            if (handler && backgroundTaskIdentifier == [handler invalidIdentifier]) {
```

Comment:
Could you please follow the coding rules of the rest of the project and put a EOL between ```if ()``` and ```{...```.